docs/content/management-bundles.md

```diff
@@ -569,11 +571,25 @@ The `"value"` field defines the value to be added or replaced. Only required for
 
 #### Limitations
```

Comment:
With your added points below, I feel like we probably want to add a "**Current** [Limitations]" here, or a few "currently" in the bullet points... just to emphasize these aren't limitations in the format, but things that have yet to be implemented :)

Reply:
I've still added Current to this title, but have moved the new item to the format section above in 2536f44b48f73ace383660522ff84c3e7b438581 as per Ash's comment.